### PR TITLE
Exchange Text

### PR DESCRIPTION
**Pull request by ntut-xuan**
Hi I do spaghetti.

 - [x] Implement exchange feature to show to currency value between ETH and TWD.

image

### CODE REVIEW

File: packages/frontend/src/routes/index.tsx [hunk 1]
```diff
@@ -23,15 +23,27 @@ type BalanceProps = {
...
+            "https://api.coinbase.com/v2/exchange-rates?currency=ETH"
+        );
+        const data = await response.json();
+        return (
+            Number.parseFloat(formatEther(value)) * data["data"]["rates"]["TWD"]
+        );
+    };
+
     const balanceString = createAsync(async () => {
+        props.address;
```
Comment (NOOBDY) [inline packages/frontend/src/routes/index.tsx:38]: this line is not required

File: packages/frontend/src/routes/index.tsx [hunk 2]
```diff
@@ -23,15 +23,27 @@ type BalanceProps = {
...
+        return `${formatEther(balance.value)} ETH (${await getExchangeData(balance.value)} TWD)`;
     });
 
     return (
-        <p class="font-mono">
-            <Suspense fallback={"loading"}>{balanceString()}</Suspense>
-        </p>
+        <Suspense>
+            <p class="font-mono">{balanceString()}</p>
+        </Suspense>
```
Comment (NOOBDY) [inline packages/frontend/src/routes/index.tsx:46]: why remove fallback text :(

**Review by NOOBDY — CHANGES_REQUESTED** [review]

File: packages/frontend/src/routes/index.tsx (hunk 2 above)
Comment (ntut-xuan) [reply packages/frontend/src/routes/index.tsx]: Fixed.

File: packages/frontend/src/routes/index.tsx (hunk 1 above)
Comment (ntut-xuan) [reply packages/frontend/src/routes/index.tsx]: Fixed.

**Review by NOOBDY — APPROVED** [review]

**Pull request merged**